internal/tools/utils/utils.go

```diff
@@ -113,6 +113,8 @@ func (r *ResourceUtil) GetDatabase(installation *model.Installation) model.Datab
 		return model.NewMysqlOperatorDatabase()
 	case model.InstallationDatabaseAwsRDS:
```

Comment:
1/5 maybe we should change this old type to `InstallationDatabaseSingleTenantRDS` for consistency with the new type?